braindecode/models/base.py

```diff
@@ -197,6 +199,18 @@ def get_output_shape(self) -> Tuple[int]:
                     raise ValueError(msg) from exc
                 raise exc
 
+    def return_new_keys(self, state_dict, keys_to_change):  # personalised keys_to_change
```

Comment:
I would also remove the comment in the end of line -> if there is sth user should know put it in the docstring